model_zoo/spleen_ct_segmentation_pytorch/README.md

```diff
@@ -0,0 +1,61 @@
+# Description
+Load pre-trained model from MONAI model-zoo with pytorch and train it on a new dataset.
+
+# Overview
+This tutorial shows how to load a pretrained U-net from MONAI model-zoo and train it on BTCV dataset using pytorch. The pretrained U-net was trained on volumetric (3D) segmentation of CT spleen from MSD dataset.
+
+## Data
+The description of data and data preparation can be found in [spleen_ct_segmentation_monai_bundle](../spleen_ct_segmentation_monai_bundle)
+
+## Pretrained model
+The pretrained model is from MONAI model-zoo spleen_ct_segmentation.
+It was trained using the runner-up [1] awarded pipeline of the "Medical Segmentation Decathlon Challenge 2018" using the UNet architecture [2] with 32 training images and 9 validation images.
+
+It is loaded with the following python code, which can be found in `train.py`.
+```
+import monai
+pretrained_model = monai.bundle.load(
+    name="spleen_ct_segmentation", bundle_dir="./", version="0.1.1"
+)
+model.load_state_dict(pretrained_model)
+```
+
+## Training configuration
+The training was performed with at least 12GB-memory GPUs.
+During training, 19 out of the 24 scans are used for training, while the rest 5 is used for validation and model selection.
+
+Actual Model Input: 96 x 96 x 96
+
+## Input and output formats
+Input: 1 channel CT image
+
+Output: 2 channels: Label 1: spleen; Label 0: everything else
+
+## Scores
+This model achieves the following Dice score on the validation data:
+
+When training with BTCV data from scratch, we got mean Dice = 0.9294.
+When finetuning with BTCV data from the pretrained model, we got mean Dice = 0.9488
+
+The Dice of finetuning result is better than training from scratch for every subject in the validation set.
+
+## Commands example
+Train and evaluate from scratch:
+```
+python3 train.py
+python3 evaluate.py
+```
+
+Load pre-trained model and do transfer learning:
+```
+python3 train.py --load_pretrained_ckpt
+python3 evaluate.py --load_pretrained_ckpt
```

Comment:
I think no need to distinguish the models during evaluation, because no matter with or without pretrained weights, the models for evaluation are trained by `train.py`. Here we just need to evaluate the trained model.

Thanks.

Reply:
Same as the comment in the evaluation. We used this option to train two models and compare the performance.